test/screenshot/golden.json

```diff
@@ -432,11 +432,11 @@
     }
   },
   "spec/mdc-list/classes/checkbox.html": {
-    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2019/05/10/14_25_41_111/spec/mdc-list/classes/checkbox.html?utm_source=golden_json",
+    "public_url": "https://storage.googleapis.com/mdc-web-screenshot-tests/travis/2020/01/09/13_35_33_056/spec/mdc-list/classes/checkbox.html?utm_source=golden_json",
```

Comment:
is this white-tick color expected on new secondary color? Can we use the same ink color on new secondary for consistency? Fab renders black on new secondary as per above screenshots.

Reply:
That's because checkbox [use secondary](https://github.com/material-components/material-components-web/blob/master/packages/mdc-checkbox/_variables.scss#L30) as a background but use [on primary](https://github.com/material-components/material-components-web/blob/master/packages/mdc-checkbox/_variables.scss#L27) for checkmark.

Should I update it?

Reply:
I think we need to fix that first. Can you change it to `on-secondary`? Also to other components wherever applicable.

Reply:
In this PR or create another Issue an another PR?

Reply:
Preferably in the same PR since this PR affects the ink color in components.

Reply:
Done!

Reply:
Also added separate style for disabled checkbox mark. Now it is the same for enabled and disabled and depends on secondary color. So change of secondary color affects disabled checkbox checkmark which is on the same grey background.